.changeset/slick-kings-travel.md

```
@@ -0,0 +1,5 @@
+---
+"ensapi": minor
+---
+
+Renamed environment variable: `DATABASE_URL` to `ENSDB_URL`.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)